### ISSUE TITLE
What is included in the data set mentioned here? The training set has 1450 images, but it seems that there are two data set files, but the test set has 5 data sets?

### ISSUE THREAD

**Issue by WangZhi-wz** [DengPingFan/PraNet#52]
What is included in the data set mentioned here? The training set has 1450 images, but it seems that there are two data set files, but the test set has 5 data sets?

下载测试数据集并将其移至./data/TestDataset/，可以在此[下载链接（Google Drive）](https://drive.google.com/file/d/1o8OfBvYE6K-EpDyvzsmMPndnUMwb540R/view?usp=sharing)中找到。

下载训练数据集并将其移至./data/TrainDataset/，可在此[下载链接 (Google Drive)](https://drive.google.com/file/d/1lODorfB33jbd-im-qrtUgWnZXxB94F55/view?usp=sharing)中找到。

**Comment by GewelsJI** [comment]
@MOONWZ 你好，

训练集合确实是混合数据集合，测试集合是分开独立进行测试验证的

此外，下载google文件后，解压后即可以使用，如果有问题可以把文件结构截图发上来帮你看看

**Comment by WangZhi-wz** [comment]
> @MOONWZ 你好，
> 
> 训练集合确实是混合数据集合，测试集合是分开独立进行测试验证的
> 
> 此外，下载google文件后，解压后即可以使用，如果有问题可以把文件结构截图发上来帮你看看

TrainDataset中有1450张图片，550张是CVC-ClinicDB，900张是Kvasir-SEG，意思就是训练数据集只使用了CVC-ClinicDB和Kvasir-SEG，而测试数据集用了五个数据集测试结果么

**Comment by GewelsJI** [comment]
是的

**Comment by WangZhi-wz** [comment]
> 是的

好的，感谢您

**Comment by WangZhi-wz** [comment]
我自己看了一下TrainDataset中有1450张图片，550张是CVC-ClinicDB，900张是Kvasir-SEG，但是TestDataset中包括CVC-300，CVC-ClinicDB，CVC-ColonDB，ETIS-LaribPolypDB和Kvasir，您在训练过程中直接使用的包括550张是CVC-ClinicDB和900张是Kvasir-SEG的TrainDataset么，没有使用其他数据集训练？还是有使用五种数据集的训练集单独训练和测试？如何可以能否提供一下CVC-300和CVC-ColonDB数据集，因为它们的官网停止访问了。






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "DengPingFan/PraNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月29日(星期一) 下午4:07
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [DengPingFan/PraNet] What is included in the data set mentioned here? The training set has 1450 images, but it seems that there are two data set files, but the test set has 5 data sets? (Issue #52)





 
你好，下载google文件后，解压后即可以使用，如果有问题可以把文件结构截图发上来帮你看看

**Comment by GewelsJI** [comment]
@MOONWZ 你好，所有的训练数据和测试数据可以在这里（[train](https://drive.google.com/file/d/1lODorfB33jbd-im-qrtUgWnZXxB94F55/view)和[test](https://drive.google.com/file/d/1o8OfBvYE6K-EpDyvzsmMPndnUMwb540R/view)）获取到，官网数据无法访问建议可以直接与原作者进行联系。此外我们是用同一个well-trained模型在所有的测试集合上进行测试，不需要额外进行单独的训练，因为我们希望PraNet的是一个通用的息肉分割模型。谢谢关注！
